rubocop.yml

```diff
@@ -50,6 +49,12 @@ Style/AccessorMethodName:
   Enabled: false
 Style/Alias:
   Enabled: false
+Style/BlockDelimiters:
+  EnforcedStyle: semantic
```

Comment:
This is *exactly* what semantic is for: it requires braces or keywords based on which methods use the block's value only (functionals) and which run the block in some context (procedural). 

Before falls in the second category, so requiring do..end is a _good_ idea.